srcs/parsing/check_map_content.c

```diff
@@ -3,10 +3,10 @@
 /*                                                        :::      ::::::::   */
 /*   check_map_content.c                                :+:      :+:    :+:   */
 /*                                                    +:+ +:+         +:+     */
-/*   By: aweaver <aweaver@student.42.fr>            +#+  +:+       +#+        */
+/*   By: elouisia <elouisia@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2022/09/16 13:06:38 by aweaver           #+#    #+#             */
-/*   Updated: 2022/09/28 14:47:00 by aweaver          ###   ########.fr       */
+/*   Updated: 2022/10/02 13:36:21 by elouisia         ###   ########.fr       */
```

Comment:
This file should not have been added because nothing was modified in it. Could you remove it please? :D

Reply:
Okay so TECHNICALLY, I fixed it using rebase -i and restore and all the space-travel related functions but I may have forgotten to disable the header auto update...